src/Cache/_ScsDataClient.php

```diff
@@ -542,4 +553,82 @@ public function dictionaryGetBatch(string $cacheName, string $dictionaryName, ar
     }
+
+    public function dictionaryIncrement(
+        string $cacheName, string $dictionaryName, string $field, bool $refreshTtl, int $amount = 1, ?int $ttlSeconds = null
+    ): CacheDictionaryIncrementResponse
+    {
+        try {
+            validateCacheName($cacheName);
+            validateDictionaryName($dictionaryName);
+            validateFieldName($field);
+            $ttlMillis = $this->ttlToMillis($ttlSeconds);
+            validateTtl($ttlMillis);
+            $dictionaryIncrementRequest = new _DictionaryIncrementRequest();
+            $dictionaryIncrementRequest
+                ->setDictionaryName($dictionaryName)
+                ->setField($field)
+                ->setAmount($amount)
+                ->setRefreshTtl($refreshTtl)
+                ->setTtlMilliseconds($ttlMillis);
+            $call = $this->grpcManager->client->DictionaryIncrement(
+                $dictionaryIncrementRequest, ["cache" => [$cacheName]], ["timeout" => $this->deadline_seconds * self::$TIMEOUT_MULTIPLIER]
+            );
+            $response = $this->processCall($call);
+        } catch (SdkError $e) {
+            return new CacheDictionaryIncrementResponseError($e);
+        } catch (Exception $e) {
+            return new CacheDictionaryIncrementResponseError(new UnknownError($e->getMessage()));
+        }
+        return new CacheDictionaryIncrementResponseSuccess($response);
+    }
+
+    public function dictionaryRemoveField(string $cacheName, string $dictionaryName, string $field): CacheDictionaryRemoveFieldResponse
+    {
+        try {
+            validateCacheName($cacheName);
+            validateDictionaryName($dictionaryName);
+            validateFieldName($field);
+            $dictionaryRemoveFieldRequest = new _DictionaryDeleteRequest();
+            $some = new _DictionaryDeleteRequest\Some();
+            $some->setFields([$field]);
+            $dictionaryRemoveFieldRequest->setDictionaryName($dictionaryName);
+            $dictionaryRemoveFieldRequest->setSome($some);
+            $call = $this->grpcManager->client->DictionaryDelete(
+                $dictionaryRemoveFieldRequest, ["cache" => [$cacheName]], ["timeout" => $this->deadline_seconds * self::$TIMEOUT_MULTIPLIER]
+            );
+            $this->processCall($call);
+        } catch (SdkError $e) {
+            return new CacheDictionaryRemoveFieldResponseError($e);
+        } catch (Exception $e) {
+            return new CacheDictionaryRemoveFieldResponseError(new UnknownError($e->getMessage()));
+        }
+        return new CacheDictionaryRemoveFieldResponseSuccess();
+    }
+
+    public function dictionaryRemoveFields(string $cacheName, string $dictionaryName, array $fields): CacheDictionaryRemoveFieldsResponse
+    {
+        try {
+            validateCacheName($cacheName);
+            validateDictionaryName($dictionaryName);
+            foreach ($fields as $field) {
+                validateFieldName($field);
+            }
```

Comment:
Do we also want to validate `$fields` array is not an empty array?

Reply:
Yes, we do! Thanks, good catch. Fixed in 6ea1091.